bosh-deployment.yml

```diff
@@ -35,6 +35,8 @@ releases:
   version: latest
 - name: uaa
   version: latest
+- name: cron,
```

Comment:
Guessing this errant comma may break things, but my review is apparently too slow for @sharms